src/handlers/keyDecryption.js

```diff
@@ -50,7 +50,7 @@ export async function handleKeyDecryption (request, invocation, ctx, env) {
     }
 
     // Step 3: Check revocation status
-    const revocationResult = await ctx.revocationStatusService?.checkStatus(invocation.proofs, env)
+    const revocationResult = await ctx.revocationStatusService?.checkStatus(proofs, request.space, env)
```

Comment:
I'm not sure you should do this seperately - Ucanto server has a method `verifyAuthorization` that is designed for this.